query-engine/black-box-tests/tests/metrics/smoke_tests.rs

```diff
@@ -0,0 +1,65 @@
+use query_engine_tests::*;
+
+/// Asserts common basics for composite type writes.
+#[test_suite(schema(schema))]
+mod smoke_tests {
+    fn schema() -> String {
+        let schema = indoc! {
+            r#"model Person {
+              #id(id, Int, @id)
+             }"#
+        };
+
+        schema.to_owned()
+    }
+
+    #[connector_test]
+    async fn expected_metrics_rendered(r: Runner) -> TestResult<()> {
+        // spwan a query-engine with metrics enabled that is configured to use the same
+        // DML as the one used by the test suite as provided by the schema function above.
+        let path = assert_cmd::cargo::cargo_bin("query-engine");
```

Comment:
This can be done without dependency, it's built into cargo (see https://github.com/prisma/prisma-engines/blob/9f8aaa085b47e997f8f789a474532622d4ab6eab/schema-engine/cli/tests/cli_tests.rs#L15 for an example in prisma-engines). I think it's the only use for assert_cmd in this crate, so we could do without the dependency.